### ISSUE TITLE
VS extension doesn't give me any session ID

### ISSUE THREAD

**Issue by wes-goulet** [MobileEssentials/FormsPlayer#1]
When I launch the VS extension it doesn't show me any session ID, it's just blank.

**Comment by wes-goulet** [comment]
I should mention that even if I click "Connect" it still doesn't give me a session id

**Comment by wes-goulet** [comment]
Also, this is on Windows 10 build 10130 with VS2015 RC

**Comment by cbumstead** [comment]
Yes, I'm having the same issue with VS2013, if I put in a matching session id I still can't connect.    Is there anything special that needs to be done?    Does it matter what project is open?

**Comment by wes-goulet** [comment]
I actually got this to work... I just had to put in some made up session ID in VS window, then click connect, then on the phone apps I put in the same session ID and hit connect and it seems to work.  I originally thought the VS plugin would give me a session ID but I guess you just make up your own.

**Comment by joagwa** [comment]
I'm not having any luck. When I create some ransom Session ID, then press connect, it freezes for a moment then does nothing. Disconnect button is still disabled. If I put the same ID in the app, and press connect, it seems to indicate that it is connected (but doesn't actually do anything). 

Actually, I just noted I can put in whatever I want in the app and it says that it is connected

EDIT: Seems to be working now. It may have been a proxy issue inside my corp network

**Comment by kzu** [comment]
Both VS and device just connect to a signalr hub in azure. That doesn't
require the other end to be connected, that's why you can connect the
device to a "session" without having VS successfully connected too.

The error seems to be on the VS side. If it's not suggesting a default
session id, that may be a symptom: I use a hash of the mac address to
generate it, so either VS doesn't have permissions to get that data, our
you are running on a VM without any configured network hardware, or
something :).

You can just grab the source and debug the extension to determine the root
cause.

**Comment by MBrekhof** [comment]
Latest release might have fixed problems. FWIW, installed the APK on the VS Android Emulator and the VSIX in VS 2015. Took a while for the player and VS to find each other but it now shows my form on the android emulator when I save the xaml in VS.
